[2018-02-07 09:09:07,399] {models.py:167} INFO - Filling up the DagBag from /home/ubuntu/workspace/airflow/dags/Main_dag.py
[2018-02-07 09:09:38,302] {timeout.py:37} ERROR - Process timed out
[2018-02-07 09:09:42,296] {models.py:266} ERROR - Failed to import: /home/ubuntu/workspace/airflow/dags/Main_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airflow/models.py", line 263, in process_file
    m = imp.load_source(mod_name, filepath)
  File "/home/ubuntu/workspace/airflow/dags/Main_dag.py", line 41, in <module>
    schedule_interval='0 1 * * *',
  File "/usr/local/lib/python2.7/dist-packages/airflow/models.py", line 2663, in __init__
    self.fileloc = inspect.getsourcefile(inspect.stack()[1][0])
  File "/usr/lib/python2.7/inspect.py", line 1060, in stack
    return getouterframes(sys._getframe(1), context)
  File "/usr/lib/python2.7/inspect.py", line 1038, in getouterframes
    framelist.append((frame,) + getframeinfo(frame, context))
  File "/usr/lib/python2.7/inspect.py", line 1013, in getframeinfo
    lines, lnum = findsource(frame)
  File "/usr/lib/python2.7/inspect.py", line 532, in findsource
    module = getmodule(object, file)
  File "/usr/lib/python2.7/inspect.py", line 497, in getmodule
    f = getabsfile(module)
  File "/usr/lib/python2.7/inspect.py", line 466, in getabsfile
    _filename = getsourcefile(object) or getfile(object)
  File "/usr/lib/python2.7/inspect.py", line 451, in getsourcefile
    if os.path.exists(filename):
  File "/usr/lib/python2.7/genericpath.py", line 26, in exists
    os.stat(path)
  File "/usr/local/lib/python2.7/dist-packages/airflow/utils/timeout.py", line 38, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
AirflowTaskTimeout: Timeout
[2018-02-07 09:27:35,381] {models.py:167} INFO - Filling up the DagBag from /home/ubuntu/workspace/airflow/dags/Main_dag.py
[2018-02-07 09:27:35,700] {base_task_runner.py:112} INFO - Running: ['bash', '-c', u'airflow run Jobsv31.doubleclick_search_keyword create_folder_with_current_date 2017-11-05T01:00:00 --job_id 168 --raw -sd DAGS_FOLDER/Main_dag.py']
[2018-02-07 09:27:35,923] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:35,923] {__init__.py:57} INFO - Using executor LocalExecutor
[2018-02-07 09:27:36,006] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,005] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-02-07 09:27:36,029] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,029] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-02-07 09:27:36,175] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,175] {models.py:167} INFO - Filling up the DagBag from /home/ubuntu/workspace/airflow/dags/Main_dag.py
[2018-02-07 09:27:36,486] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,486] {models.py:1126} INFO - Dependencies all met for <TaskInstance: Jobsv31.doubleclick_search_keyword.create_folder_with_current_date 2017-11-05 01:00:00 [queued]>
[2018-02-07 09:27:36,490] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,490] {models.py:1126} INFO - Dependencies all met for <TaskInstance: Jobsv31.doubleclick_search_keyword.create_folder_with_current_date 2017-11-05 01:00:00 [queued]>
[2018-02-07 09:27:36,490] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,490] {models.py:1318} INFO - 
[2018-02-07 09:27:36,490] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-02-07 09:27:36,490] {base_task_runner.py:95} INFO - Subtask: Starting attempt 1 of 4
[2018-02-07 09:27:36,491] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-02-07 09:27:36,491] {base_task_runner.py:95} INFO - Subtask: 
[2018-02-07 09:27:36,508] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,508] {models.py:1342} INFO - Executing <Task(BashOperator): create_folder_with_current_date> on 2017-11-05 01:00:00
[2018-02-07 09:27:36,555] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,555] {bash_operator.py:71} INFO - tmp dir root location: 
[2018-02-07 09:27:36,555] {base_task_runner.py:95} INFO - Subtask: /tmp
[2018-02-07 09:27:36,556] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,556] {bash_operator.py:80} INFO - Temporary script location :/tmp/airflowtmpXUlI9s//tmp/airflowtmpXUlI9s/create_folder_with_current_date3OBBq4
[2018-02-07 09:27:36,556] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,556] {bash_operator.py:81} INFO - Running command: /home/ubuntu/workspace/airflow/bash/create_folder_with_current_date.sh doubleclick_search 20171105
[2018-02-07 09:27:36,560] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,560] {bash_operator.py:90} INFO - Output:
[2018-02-07 09:27:36,562] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,561] {bash_operator.py:94} INFO - ================test!!!!!!!!==========
[2018-02-07 09:27:36,566] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,565] {bash_operator.py:94} INFO - Pseudo-terminal will not be allocated because stdin is not a terminal.
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,761] {bash_operator.py:94} INFO - Welcome to Ubuntu 16.04.3 LTS (GNU/Linux 4.4.0-1041-aws x86_64)
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,761] {bash_operator.py:94} INFO - 
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - * Documentation:  https://help.ubuntu.com
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - * Management:     https://landscape.canonical.com
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - * Support:        https://ubuntu.com/advantage
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - Get cloud support with Ubuntu Advantage Cloud Guest:
[2018-02-07 09:27:36,762] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - http://www.ubuntu.com/business/services/cloud
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 71 packages can be updated.
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 0 updates are security updates.
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - 
[2018-02-07 09:27:36,763] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,762] {bash_operator.py:94} INFO - *** System restart required ***
[2018-02-07 09:27:36,765] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,765] {bash_operator.py:94} INFO - ========executing this file in ETL!!! ===============
[2018-02-07 09:27:36,766] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,765] {bash_operator.py:94} INFO - /home/ubuntu/csv_files/doubleclick_search/20171105
[2018-02-07 09:27:36,768] {base_task_runner.py:95} INFO - Subtask: [2018-02-07 09:27:36,767] {bash_operator.py:97} INFO - Command exited with return code 0
[2018-02-07 09:27:40,704] {jobs.py:2083} INFO - Task exited with return code 0
